### ISSUE TITLE
File not found - app | installation unsuccessful 

### ISSUE THREAD

**Issue by sergzoer** [Swirt/typertools-src#71]
Hi! I have tried installing Typertools onto Photoshop 2022 and have received this error message in the cmd interface:
<img width="272" alt="Screenshot 2024-04-23 174805" src="https://github.com/Swirt/typertools-src/assets/22253383/99f60b11-856f-4d4c-be53-d0b95a2c763f">
After which, the extension does not work on photoshop. I can still see it but it also shows an error message: 
![image_2024-04-23_175554304](https://github.com/Swirt/typertools-src/assets/22253383/d3912766-5eed-45ac-a671-b8edd972bb31)

Does anyone know what went wrong, or how to fix this issue?

**Comment by Swirt** [comment]
Did you unzip the archive before installing?
If that is not the reason, you can try manually copying the files from the archive to the folder specified in the error message.

**Comment by sergzoer** [comment]
Yep I have unzipped it into my desktop and ran the install_win from there and got this message.
Also, the weird thing is that the folder specified in the error (CEP/extensions/typertools) doesn't exist so I'm unsure where I could manually copy the files to?
![image_2024-04-23_212834979](https://github.com/Swirt/typertools-src/assets/22253383/03b3c338-472a-488a-99f5-1d4142c15643)
Is there a way to manually move the files from the zip to photoshop, or which files would I need to move?

**Comment by Swirt** [comment]
Hmm. Try manually creating the path `C:/Users/Asus/AppData/Roaming/Adobe/CEP/extensions/typertools`, and put the `app`, `CSXS`, `icons`, `locale` folders from the archive inside it. So that the file from the error message exists.

**Comment by Ryujin1108Scans** [comment]
> Hi! I have tried installing Typertools onto Photoshop 2022 and have received this error message in the cmd interface: <img alt="Screenshot 2024-04-23 174805" width="272" src="https://private-user-images.githubusercontent.com/22253383/324803393-99f60b11-856f-4d4c-be53-d0b95a2c763f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUwNTcwNDYsIm5iZiI6MTczNTA1Njc0NiwicGF0aCI6Ii8yMjI1MzM4My8zMjQ4MDMzOTMtOTlmNjBiMTEtODU2Zi00ZDRjLWJlNTMtZDBiOTVhMmM3NjNmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjI0VDE2MTIyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM1YjkyZWJlMDk4NDZmYmEwZTQ0MDA4ZjZmZmY1ZDZhOTljZGQ2MTc3NzU2MjZhODMxM2U4NjQ4M2NiN2RjNmUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.4ooHR0T_C10dSTQCq1luon7wiZ_YeIwQ6SqSCX3xias"> After which, the extension does not work on photoshop. I can still see it but it also shows an error message: ![image_2024-04-23_175554304](https://private-user-images.githubusercontent.com/22253383/324803678-d3912766-5eed-45ac-a671-b8edd972bb31.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUwNTcwNDYsIm5iZiI6MTczNTA1Njc0NiwicGF0aCI6Ii8yMjI1MzM4My8zMjQ4MDM2NzgtZDM5MTI3NjYtNWVlZC00NWFjLWE2NzEtYjhlZGQ5NzJiYjMxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjI0VDE2MTIyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZhNGQ4ZGZlODdmM2NlMDJlYTA3NzNhMzVjY2Q2MGNmMThhMWFkMGQ0NWQ0OWU1ZWIxODU4OTM5YjQ1NTE3ZDgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.o8na8Hw-q2Vfc2PSAbnzKEuMV5uuxb5XH_xa2POpJPo)
> 
> Does anyone know what went wrong, or how to fix this issue?

The error is due to the installation from the github's source file zip. you had to download from the site. also about why you are not able to find the path is because you are trying to search it from the normal explorer whereas you had to go to the path using the run dialog box which you can get from clicking `Win+R` and enter `%appdata%\Adobe` to get the adobe folder in the roaming path.

**Comment by lifefk** [comment]
FOR BLANK, GREY, OR WHITE TYPETOOL with no content

1. Download Typetool of any version 
2. Extract using WinRAR to a folder named Typetool, make sure the version number is erased from the folder name
3. Copy the folder and paste it at Local Disk(C:) > Program Files > Adobe > Adobe Photoshop 2023 > Required > CEP >extensions
*Same process for any version of Photoshop
*Photoshop must be closed during this process.
